reconftw.sh

```diff
@@ -1684,6 +1685,25 @@ function prototype_pollution(){
 	fi
 }
 
+###############################################################################################################
+######################################### VULNERABILITIES #####################################################
+###############################################################################################################
+
+function lgtm(){
+	if { [ ! -f "$called_fn_dir/.${FUNCNAME[0]}" ] || [ "$DIFF" = true ]; } && [ "$LGTM_CLI" = true ] ; then
+		start_func ${FUNCNAME[0]} "CodeQL scan"
+		# TODO
+		# lgtm --ignore-followed-errors --nocache --wait 30s follow -f REPO_LIST
+		# by code search?
```

Comment:
Would it be useful to add a code search function? lgtm-cli lets you specify strings and it will search for them on github code search API to follow the resulting repositories.